perf/benchmark.jl

```diff
@@ -20,30 +22,49 @@ function benchmark_driver!(f, x...; f_displayname=string(f))
     print("  Run TapedFunction (compiled):")
     @btime $ctf($(x)...)
     GC.gc()
+
+    print("  Run TapedTask: ")
+    x = (x[1:end-1]..., produce);
+    # show the number of produce calls inside `f`
+    tf = Libtask.TapedFunction(f, x...);
+    tt = TapedTask(tf, x...); 
+    c = 0; while consume(tt)!==nothing c+=1 end;
```

Comment:
Also non-idiomatic Julia code.

```suggestion
    c = 0
    while consume(tt) !== nothing
        c += 1
    end
```